tests/namespaced.rs

```diff
@@ -12,697 +12,701 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use bytes::Bytes;
-use futures::future::poll_fn;
-use http_body_util::Empty;
-use std::collections::HashMap;
-use std::net::{IpAddr, SocketAddr};
-use std::str::FromStr;
-use std::time::Duration;
-
-use hyper::Method;
-use tokio::io::{AsyncReadExt, AsyncWriteExt, ReadBuf};
-use tokio::net::TcpStream;
-use tokio::time::timeout;
-use tracing::{error, info};
-
-use ztunnel::identity;
-use ztunnel::test_helpers::app::ParsedMetrics;
-use ztunnel::test_helpers::app::TestApp;
-use ztunnel::test_helpers::components::WorkloadManager;
-use ztunnel::test_helpers::helpers::initialize_telemetry;
-use ztunnel::test_helpers::netns::{Namespace, Resolver};
-use ztunnel::test_helpers::*;
-
-macro_rules! function {
-    () => {{
-        fn f() {}
-        fn type_name_of<T>(_: T) -> &'static str {
-            std::any::type_name::<T>()
-        }
-        let name = type_name_of(f);
-        &name[..name.len() - 3]
-    }};
-}
+#[cfg(all(test, target_os = "linux"))]
```

Comment:
FYI nothing actually changed in this file apart from adding a surrounding `mod namespaced` so I can easily enable only for linux.